src/components/AppMenu/TabBar.vue

```diff
@@ -3,11 +3,48 @@
       <div id="tabNavigation">
         <div class="systemButton historyButton" @click="historyBack" @click.middle="historyBackNewTab" :disabled="!activeTabHasHistory"><fa-icon icon="chevron-left"></fa-icon></div>
         <div class="systemButton historyButton" @click="historyForward" @click.middle="historyForwardNewTab" :disabled="!activeTabHasFuture"><fa-icon icon="chevron-right"></fa-icon></div>
+
+        <div class="systemButton historyButton" @click="reloadTab" @click.middle="forceReload" style="font-size: 21px;"><fa-icon icon="redo"></fa-icon></div>
       </div>
       <div id="jumpBox">
         <div class="jumpBoxWrapper">
           <a :href="jumpboxText" class="jumpboxLink" ref="link" />
-          <input class="jumpBoxInput" ref="input" type="text" spellcheck="false" v-model="jumpboxText" @keydown.esc="resetJumpbox()" @keydown.enter="focusLink()" />
+          <vue-simple-suggest
+            v-model="jumpboxText" 
+            :list="allUrlSuggestions"
+            display-attribute="title"
+            value-attribute="url"
+            @select="onSuggestSelect"
+            :filter-by-query="true"
+            :max-suggestions="15"
+            ref="suggest" 
+          >
+            <input 
+              class="jumpBoxInput" 
+              ref="input" 
+              type="text" 
+              spellcheck="false" 
+              v-model="jumpboxText" 
+              @keydown.enter="focusLink"
+              @keydown.esc="resetJumpbox" 
+            />
+          </vue-simple-suggest>    
+          <div id="browserActions">
+            <div class="systemButton"
+              v-if="$localData.settings.devMode && thisTabPageId" 
+              :title="`Change current page from ${$newReaderCurrent} to p=${thisTabPageId}\n(Middle click to disable newreader)`"
+              :class="{active: thisTabPageId != $newReaderCurrent}"
+              @click="$updateNewReader(thisTabPageId, true)" 
+              @click.middle="$localData.root.NEW_READER_CLEAR">
+              <fa-icon icon="lock"></fa-icon>
+              <span class="badge" v-text="$newReaderCurrent"></span>
+            </div>
+            <div class="systemButton"
+              @click="isCurrentPageBookmarked ? tabComponent.$refs.bookmarks.open() : tabComponent.$refs.bookmarks.newSave()"
```

Comment:
Maybe make this one a click-to-toggle? Currently does nothing if you try to click while the menu is open

Reply:
It bookmarks the current page if it isn't already. It also opens the bookmarks pane if it isn't open already.